styleguide/_themes/global/scss/forms.scss

```diff
@@ -228,3 +228,232 @@ table {
     }
   }
 }
+
+// exposed form overrides
+$exposed-form-bg: #003482;
+
+// sass-lint:disable-block no-ids
+#block-webform {
```

Comment:
Can we change this to: `.rs-webform-embedded`?

I was able to tweak the HTML to this: 
```
<div class="rs-webform-block rs-webform-embedded" id="block-webform-block">
```

That `.rs-webform-embedded` class is specific to this particular block and you can see it's at the same level as that `id`.